src/main/java/creativei/service/impl/InquiryServiceImpl.java

```diff
@@ -100,4 +100,12 @@ public Inquiry update(Inquiry inquiry) throws  DataIntegrityException, InvalidPa
     public Inquiry updateAll(List<Inquiry> inquiries) {
         return null;
     }
+
+    @Override
+    public List<Inquiry> getUnattendedInquiry(String boolParam, String statusParam) {
+        Boolean isAttended;
+        isAttended=boolParam.equals(false)?false:true;
```

Comment:
boolParam is a string compare it with string with case ignore

Reply:
issue resolve in  aac45ec